demo-extended/src/main/java/com/novoda/downloadmanager/demo/extended/batches/BatchDownloadsActivity.java

```diff
@@ -67,6 +73,18 @@ public void onClick(@NonNull View v) {
                 });
 
         setupQueryingExample();
+
+        ConnectivityManager connectivityManager = (ConnectivityManager) getSystemService(Context.CONNECTIVITY_SERVICE);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
Any other version will use the connectivity change broadcast that's still available